torch/lib/c10d/test/FileStoreTest.cpp

```diff
@@ -10,6 +12,18 @@
 #include <c10d/FileStore.hpp>
 #include <c10d/PrefixStore.hpp>
 
+#ifdef _WIN32
+std::string tmppath() {
```

Comment:
This function looks very similar to the one you've implemented in TestUtils.h, I wonder if they could be merged.